docs/root/intro/arch_overview/http/http3.rst

```diff
@@ -0,0 +1,40 @@
+.. _arch_overview_http3:
+
+HTTP3 downstream
+================
+
+HTTP/3 downstream support is still in Alpha, and should be used with caution.
+Outstanding issues required for HTTP/3 to go GA can be found
+`here <https://github.com/envoyproxy/envoy/labels/quic-mvp>`_
+
+Envoy HTTP/3 support can be turned up via adding
+:ref:`quic_options <envoy_v3_api_field_config.listener.v3.UdpListenerConfig.quic_options>` and
+ensuring the downstream transport socket is a QuicDownstreamTransport.
+
+See example :repo:`downstream HTTP/3 configuration </configs/envoyproxy_io_proxy_http3_downstream.template.yaml1>` for example configuration.
+
+Note that the example configuration includes both a TCP and a UDP listener, and the TCP
+listener is advertising http/3 support via an alt-svc header. Advertising HTTP/3 is not necessary for
```

Comment:
`alt-svc`

Reply:
i think this is probably more important as it wants to emphasize the is the exact header